src/Console/Shell.php

```diff
@@ -163,6 +163,7 @@ public function __construct(ConsoleIo $io = null)
 
         $locator = $this->tableLocator() ? : 'Cake\ORM\TableRegistry';
         $this->modelFactory('Table', [$locator, 'get']);
+        $this->modelType('Table');
```

Comment:
This change is meant for 3.2 because that.
On 24 Oct 2015 8:19 p.m., "Mark Story" <notifications@github.com> wrote:

> In src/Console/Shell.php
> <https://github.com/cakephp/cakephp/pull/7587#discussion_r42936642>:
>
> > @@ -163,6 +163,7 @@ public function __construct(ConsoleIo $io = null)
> >
> >          $locator = $this->tableLocator() ? : 'Cake\ORM\TableRegistry';
> >          $this->modelFactory('Table', [$locator, 'get']);
> > +        $this->modelType('Table');
>
> Needing to call this method makes this change backwards incompatible.
> Including the trait no longer gives you access to the ORM locator anymore
> by default - which it probably should to be more backwards compatible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/cakephp/pull/7587/files#r42936642>.
>